### PR TITLE
Integrate Floor Intake subsystem

### PR DESCRIPTION
**Pull request by jagernet-ops**
Created two associated commands to the FloorIntake subsystem which control the direction of the motor. The commands are mapped to the X and A buttons on the operator controller.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,2 @@
 # frc1675-2023
-Team 1675 robot code for the 2023 FRC competition Charge Up
-
-Justin was here
-Kai is here
-stella infernus maximus 
-Gremlin
-cheezeits suck 
+Floor Intake Git Branch Code
```
Comment (jdlanning) [inline README.md]: Please revert README!!!

File: src/main/java/frc/robot/subsystems/FloorIntake.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  public void intakePowerKill(){
+    intakeMotor.set(0);
+  }
+
+  
+  public void intakeInwards(double speed){
+    intakeMotor.set(speed*-1);
+  }
+
+  public void intakeOutput(double speed){
```
Comment (jdlanning) [inline src/main/java/frc/robot/subsystems/FloorIntake.java]: Lets us the same naming as the other Intake subsystem ... so like drop, pickup and stop but no need to method cube and cone

File: src/main/java/frc/robot/commands/FloorIntakeOut.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+import frc.robot.Constants;
+import frc.robot.subsystems.FloorIntake;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class FloorIntakeOut extends CommandBase {
```
Comment (jdlanning) [inline src/main/java/frc/robot/commands/FloorIntakeOut.java]: Lets call FloorDrop

File: src/main/java/frc/robot/commands/FloorIntakeIn.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+import frc.robot.subsystems.FloorIntake;
+import frc.robot.Constants;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class FloorIntakeIn extends CommandBase {
```
Comment (jdlanning) [inline src/main/java/frc/robot/commands/FloorIntakeIn.java]: FloorPickup name suggested

File: src/main/java/frc/robot/RobotContainer.java [hunk 5]
```diff
@@ -9,32 +9,46 @@
...
   private final DrivetrainSubsystem drivetrainSubsystem = new DrivetrainSubsystem();
   private final JoystickModification mod = new JoystickModification();
-  private final Joystick operatorController = new Joystick(Constants.OPERATOR_CONTROLLER);
+
+  /* 
+   * Ground Intake Controller Operation:
+   * -  Push Button once, start the intake motor and set to suck inwards
+   * -  Push the button at anytime during the live state to switch directions
+   * -  Long press the button to shut down the motor
+  */
```
Comment (jdlanning) [inline src/main/java/frc/robot/RobotContainer.java]: This comment is a nice idea but lets wait until all of our controls are mapped in more final state and we could add a nice documentation comment describing our mapping

File: README.md (hunk 1 above)
Comment (jagernet-ops) [reply README.md]: I thought I fixed it the night of change?

File: src/main/java/frc/robot/subsystems/FloorIntake.java [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class FloorIntake extends SubsystemBase {
+  /** Creates a new FloorIntake. */
+  private CANSparkMax intakeMotor = new CANSparkMax( Constants.FLOOR_INTAKE_MOTOR, MotorType.kBrushless);
+  public FloorIntake() {
+
+  }
+
+  public void intakePower(){
```
Comment (jdlanning) [inline src/main/java/frc/robot/subsystems/FloorIntake.java]: intakeStop

File: src/main/java/frc/robot/RobotContainer.java (hunk 5 above)
Comment (jdlanning) [reply src/main/java/frc/robot/RobotContainer.java]: Suggest to remove this comment until a later time after we finalize our control mapping

Comment (jdlanning) [reply src/main/java/frc/robot/RobotContainer.java]: This comment is still here, I like the idea but lets do this at a later time once our controls are finalized

**Review by jdlanning — APPROVED** [review]

**Pull request merged**